public/css/list/redundancy-group-list-item.less

```diff
@@ -0,0 +1,7 @@
+.redundancy-group-list-item {
+  .caption {
+    .object-statistics {
+      justify-content: end;
```

Comment:
You don't have it under control whether the object statistics are a flex container. The caption is which is deciding that the statistics should be aligned right, so it's responsible anyway.